README.md

```diff
@@ -180,7 +180,7 @@ Both `.yaml.erb` and `.yml.erb` file extensions are supported. Templates must re
 
 ##### Limitations when using partials
 
-Partials can be included almost everywhere in ERB templates, with one notable exception: you cannot use a partial to define a subset of fields. For example, given a partial `p` defining two fields 'a' and 'b',
+Partials can be included almost everywhere in ERB templates, with one notable exception: you must use [YAML merge keys](http://yaml.org/type/merge.html) to define a subset of fields. For example, given a partial `p` defining two fields 'a' and 'b',
```

Comment:
Thanks for opening this PR. I didn't know this was a thing and we should definitely add it to the readme. What do you think of this slight edit:

```
... ERB templates. Note: when using a partial to insert additional key-value pairs to a map you must use [YAML merge keys](http://yaml.org/type/merge.html). For example...
```

Reply:
Sure, makes sense.